### DISCUSSION TITLE
Crop streaming and motion detection

### DISCUSSION THREAD

**Discussion by adolfossl** [Motion-Project/motion#1457] [Q&A]
0. Reviewed guide and contributing documents? (Yes/No): YES
1. version [x.y.z, hash, other]:  4.3.2
2. installed as a package or compiled from sources [deb, rpm, git, other]: deb
3. standalone or part of third party [motion, MotionEyeOS, other]: motion
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: netcam
5. hardware [x86, ARM, other]: ARM
6. operating system [16.04, Stretch, etc, FreeBSD, other]: Strech

Hello everybody.
Would it be possible to crop the image in streaming and in motion captures?
I have a peephole camera with areas that are not of interest.
The camera is an esp32S cam controlled by a raspberry pi 4 via netcam.

Thanks greetings

**Comment by tosiara** [comment]
No, motion does not have ability to crop image at runtime. You can only use a mask file to skip motion detection in the area or "censor" an area: https://motion-project.github.io/motion_config.html#mask_privacy
